src/eventobserver/EventObserver.ts

```diff
@@ -0,0 +1,13 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import EventAttributes from '../eventcontroller/EventAttributes';
+import EventName from '../eventcontroller/EventName';
+
+export default interface EventObserver {
+  /**
+   * Called when specific events occur during the meeting and includes attributes of the event. This can be used to
+   * create analytics around meeting metric.
```

Comment:
Is 'metrics' the right word here? We don't want to confuse builders with that callback.

Reply:
I think `metrics` is the right word, but I do think we can be a bit more explicit that analytics are just one of the different use cases and that it's a example so I updated the comment

Reply:
With current documentation at a first glance I immediately think that this function has some synergy with `AudioVideoObserver.metricsDidReceive` (which it doesn't) if my comment wasn't clear. I still have that thought but maybe its only me :P.